app/controllers/AppealCaseController.scala

```diff
@@ -36,6 +36,9 @@ class AppealCaseController @Inject() (
   verify: RequestActions,
   override val caseService: CasesService,
   override implicit val config: AppConfig,
+  val appeal_choose_status: views.html.appeal_choose_status,
```

Comment:
you can import `views.html` and remove from the parameters. Please do in all places wherever we have `views.html` more than once in a file